### PR TITLE
[ko] update ko team members and channels

### PR DESCRIPTION
**Pull request by hochan222**
### Description

update ko team memebers and channels 

- **ko team memeber**
  - @jho2301, @swimjiy, @cos18, @pje1740  is no longer an active member. 
  Thank you for your hard work. 고생 정말 많으셨습니다!
- **Channel**
  - add "Google Groups (yari-content-ko)"
  - remove "Kakao Talk (#MDN Korea)"

cc. @mdn/yari-content-ko

### Related issues and pull requests

- https://github.com/mdn/content/pull/33877
- https://github.com/mdn/content/pull/34188

### CODE REVIEW

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ l10n:
 
 ### 韩语（ko）
 
-- 讨论组：[Discord (#korean channel)](/discord)、[Kakao Talk (#MDN Korea)](https://open.kakao.com/o/gdfG288c)
-- 目前的志愿者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[cos18](https://github.com/cos18)、[pje1740](https://github.com/pje1740)、[wisedog](https://github.com/wisedog)、[swimjiy](https://github.com/swimjiy)、[jho2301](https://github.com/jho2301)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
+- 讨论组：[Discord (#korean channel)](/discord)、[Google Groups (`yari-content-ko`)](https://groups.google.com/g/yari-content-ko)
+- 目前的志愿者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[wisedog](https://github.com/wisedog)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:45]:
```suggestion
- 目前的志愿者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[wisedog](https://github.com/wisedog)、[jho2301](https://github.com/jho2301)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
```

File: files/zh-tw/mdn/community/contributing/translated_content/index.md [hunk 2]
```diff
@@ -41,8 +41,8 @@ l10n:
 
 ### 韓文（`ko`）
 
-- 討論群組：[Discord（`#korean`）](/discord)、[Kakao Talk（`#MDN Korea`）](https://open.kakao.com/o/gdfG288c)
-- 目前的維護者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[cos18](https://github.com/cos18)、[pje1740](https://github.com/pje1740)、[wisedog](https://github.com/wisedog)、[swimjiy](https://github.com/swimjiy)、[jho2301](https://github.com/jho2301)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
+- 討論群組：[Discord（`#korean`）](/discord)、[Google Groups (`yari-content-ko`)](https://groups.google.com/g/yari-content-ko)
+- 目前的維護者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[wisedog](https://github.com/wisedog)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
```
Comment (yin1999) [inline files/zh-tw/mdn/community/contributing/translated_content/index.md:45]:
```suggestion
- 目前的維護者：[hochan222](https://github.com/hochan222)、[yechoi42](https://github.com/yechoi42)、[wisedog](https://github.com/wisedog)、[jho2301](https://github.com/jho2301)、[sunhpark42](https://github.com/sunhpark42)、[1ilsang](https://github.com/1ilsang)
```

**Review by yin1999 — COMMENTED** [review]
It seeams that one team member is removed accidentally (according to the PR in mdn/content)

**Review by mfuji09 — APPROVED** [review]
It seems good for `ja`!

File: files/zh-cn/mdn/community/contributing/translated_content/index.md (hunk 1 above)
Comment (hochan222) [reply files/zh-cn/mdn/community/contributing/translated_content/index.md:45]: @yin1999 He is no longer an active member. 
I should have written in detail in the description, I'm sorry. I'll write it down.

Comment (hochan222) [reply files/zh-cn/mdn/community/contributing/translated_content/index.md:45]: cc. https://github.com/mdn/content/pull/34188#issuecomment-2172346477

File: files/ko/mdn/community/contributing/translated_content/index.md [hunk 3]
```diff
@@ -9,8 +9,8 @@ slug: MDN/Community/Contributing/Translated_content
 
 > **참고:** 한국어 이외의 로케일 목록은 [영어 문서](/ko/docs/MDN/Community/Contributing/Translated_content)를 참고하세요.
 
-- 토론: [Discord (#korean 채널)](/discord), [카카오톡 (MDN Korea 오픈채팅)](https://open.kakao.com/o/gdfG288c)
-- 현재 기여자: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [cos18](https://github.com/cos18), [pje1740](https://github.com/pje1740), [wisedog](https://github.com/wisedog), [swimjiy](https://github.com/swimjiy), [jho2301](https://github.com/jho2301), [sunhpark42](https://github.com/sunhpark42), [1ilsang](https://github.com/1ilsang)
+- 토론: [Discord (#korean 채널)](/discord), [Google Groups (`yari-content-ko`)](https://groups.google.com/g/yari-content-ko)
+- 현재 기여자: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [wisedog](https://github.com/wisedog), [jho2301](https://github.com/jho2301), [sunhpark42](https://github.com/sunhpark42), [1ilsang](https://github.com/1ilsang)
```
Comment (hochan222) [inline files/ko/mdn/community/contributing/translated_content/index.md:13]:
```suggestion
- 현재 기여자: [hochan222](https://github.com/hochan222), [yechoi42](https://github.com/yechoi42), [wisedog](https://github.com/wisedog), [sunhpark42](https://github.com/sunhpark42), [1ilsang](https://github.com/1ilsang)
```

File: docs/ko/guides/organizer-guide.md [hunk 4]
```diff
@@ -24,17 +24,17 @@
 - [@GwangYeol-Im](https://github.com/GwangYeol-Im) 2021.04 ~ 2023.10
 - [@yujo11](https://github.com/yujo11) 2021.04 ~ 2023.09
 - [@nKiNk](https://github.com/nKiNk) 2021.04 ~ 2023.01
+- [@pje1740](https://github.com/pje1740) 2021.04 ~ 2024.04
+- [@swimjiy](https://github.com/swimjiy) 2023.03 ~ 2024.04
+- [@cos18](https://github.com/cos18) 2021.04 ~ 2024.04
+- [@jho2301](https://github.com/jho2301) 2023.03 ~
```
Comment (hochan222) [inline docs/ko/guides/organizer-guide.md:30]:
```suggestion
- [@jho2301](https://github.com/jho2301) 2023.03 ~ 2024.05
```

**Review by irvin — APPROVED** [review]

**Review by JuanVqz — APPROVED** [review]
LGTM, ES team

**Review by leon-win — APPROVED** [review]

**Comment by leon-win** [comment]
Good for `ru` :ok_hand:

**Review by pepelsbey — APPROVED** [review]
Looks good :) Thanks!

**Review by yin1999 — APPROVED** [review]
Thank you @hochan222

**Review by cw118 — APPROVED** [review]
Looks good for l10n-fr!

**Review by tristantheb — APPROVED** [review]

**Comment by hochan222** [comment]
Thank you all!

**Pull request merged**